includes/MediaWiki/Api/EditEndpoint.php

```diff
@@ -55,19 +57,32 @@ public function run() {
 		// TODO output an object that controls the reconciliations spec?
 		$inputReconcile = $request->reconcile();
 		if ( $inputReconcile === null ) {
-			die( 'Invalid reconcile JSON supplied' );
+			throw new LocalizedHttpException(
+				MessageValue::new( 'wikibasereconcileedit-editendpoint-invalid-reconcile-json' ),
+				400
+			);
 		}
+		$supportedReconciliationVersions = [ '0.0.1' ];
```

Comment:
Not really a part of this task but i could see this moving to some other place soon.